api/FileException.json

```diff
@@ -0,0 +1,54 @@
+{
+  "api": {
+    "FileException": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/FileException",
+        "support": {
+          "chrome": {
+            "version_added": "13",
+            "prefix": "webkit"
```

Comment:
I did some more investigation here and I think we can add "version_removed": "29".
It was removed from Chrome in this issue https://bugs.chromium.org/p/chromium/issues/detail?id=251285 which dates back to June 2013, so I think it must have been removed in Chrome 29 or 30.